Kernel/VM/MemoryManager.cpp

```diff
@@ -131,17 +136,16 @@ void MemoryManager::initialize_paging()
         for (size_t page_base = mmap->addr; page_base < (mmap->addr + mmap->len); page_base += PAGE_SIZE) {
             auto addr = PhysicalAddress(page_base);
 
-            if (page_base < 7 * MB) {
-                // nothing
-            } else if (page_base >= 7 * MB && page_base < 8 * MB) {
+            // Anything below 1 * MB is a Kernel Physical region
+            if (page_base > PAGE_SIZE && page_base < 1 * MB) {
                 if (region.is_null() || !region_is_super || region->upper().offset(PAGE_SIZE) != addr) {
                     m_super_physical_regions.append(PhysicalRegion::create(addr, addr));
                     region = m_super_physical_regions.last();
                     region_is_super = true;
                 } else {
                     region->expand(region->lower(), addr);
-                }
-            } else {
+                }                
```

Comment:
Strange whitespace here.

Reply:
Looks fine in my editor, but I've run it through `clang-format` again. If it's still there I'll see what it looks like in vim.

Reply:
Okay that got rid of it